src/modules/transfer/components/AddContactModal.vue

```diff
@@ -85,30 +131,32 @@ export default{
     // });
 
     const SaveContact = () => {
-      let added = appStore.saveContact(contactName.value, address.value);
-      if(added===true){
-        err.value = '';
-        isSuccessAdded.value = true;
-        setTimeout(() => {
-          contactName.value = '';
-          emitter.emit("CLOSE_MODAL", false);
-          isAddContactQues.value = true;
-          isSuccessAdded.value = false;
-        }, 2500);
-      }else{
-        err.value = added;
-      }
+      let added = walletState.currentLoggedInWallet.addAddressBook(
+        new AddressBook(contactName.value, address.value)
+      );
```

Comment:
please check if name and address already exist, before adding. addAddressBook does not remove success status, it will always be success